administration/package.json

```diff
@@ -27,8 +28,8 @@
     "date-fns": "^2.30.0",
     "graphql": "^16.8.1",
     "localforage": "^1.10.0",
-    "normalize.css": "^8.0.1",
     "normalize-strings": "^1.1.1",
+    "normalize.css": "^8.0.1",
```

Comment:
Easiest solution would be to remove the `normalize-strings` dependency altogether :D I think, `str.normalize('NFKD').replace(/[^\x00-\x7F]/g, "")` would be a good enough replacement for it (this decomposes umlauts etc. to unicode "base" character + modifier symbol and removes non ascii characters -- such as the modifier symbols).

Reply:
Interesting approach to get rid of this problem haha, I would have spent hours at trying to set the sorting order someway. I think this is a good idea (also since the library did not receive an update in 4 years). I will add an issue to do the same for integreat.